### ISSUE TITLE
Is anyone can pretrain a 0.5b model?

### ISSUE THREAD

**Issue by ILG2021** [voicepowered-ai/VibeVoice-finetuning#20]
The inference speed is still a problem.

**Comment by jpgallegoar-vpai** [comment]
Even if you have access to millions of hours of audio like Microsoft did, this is a huge undertaking

**Comment by ILG2021** [comment]
> Even if you have access to millions of hours of audio like Microsoft did, this is a huge undertaking

I am a poor man who only has a 5090 and 100 hours. 😂

I will continue to test 1.5b and 7b, so far vibevoice hasn't yet demonstrated a huge advantage over F5 TTS, I'm hesitant about switching to Vibevoice. Because F5 has a much faster inference and higher quality, just lack text context understand.
